cirq/vis/heatmap.py

```diff
@@ -268,23 +272,28 @@ def _plot_on_axis(self, ax: plt.Axes, **collection_options: Any) -> mpl_collecti
         return collection
 
     def plot(
-        self, ax: Optional[plt.Axes] = None, **collection_options: Any
+        self, ax: Optional[plt.Axes] = None, **kwargs: Any
     ) -> Tuple[plt.Axes, mpl_collections.Collection]:
         """Plots the heatmap on the given Axes.
         Args:
             ax: the Axes to plot on. If not given, a new figure is created,
                 plotted on, and shown.
-            collection_options: keyword arguments passed to mcoll.PolyCollection().
```

Comment:
this is a breaking change technically, as `collection_options` was part of `Heatmap.plot()` in the previous releases as well. Given that we anticipate low usage on this though, I'm fine with it, but let's add a "breaking change" release note to the description of the PR and add the breaking change label.

Reply:
done.